drizzlepac/haputils/catalog_utils.py

```diff
@@ -1414,111 +1419,165 @@ def identify_sources(self, **pars):
+                        log.info("With alternate background...using RickerWavelet2DKernel to generate an alternate segmentation map.")
+
+                        # Detect segments and evaluate the detection in terms of big islands or crowded fields
+                        # Round 2
+                        ncount += 1
+                        del rw_segm_img
+                        rw_segm_img, rw_is_big_crowded, rw_bs, rw_sf = self.detect_and_eval_segments(imgarr,
+                                                                                                     rw2d_kernel,
+                                                                                                     ncount,
+                                                                                                     self._size_source_box, 
+                                                                                                     rw2d_sigma_for_threshold,
+                                                                                                     self.image.bkg_background_ra,
+                                                                                                     self.image.bkg_rms_ra,
+                                                                                                     check_big_island_only=True)
+
+                        # Last chance - check if the RickerWavelet segmentation image still seems to be problematic
+                        #
+                        # If problematic, BUT the big island and/or source fraction are below the pre-defined limits,
+                        # allow deblending as statistically these images can be deblended in a reasonable time.
+                        #
+                        # Use the second round RickerWavelet segmentation image
+                        if not rw_is_big_crowded or (rw_is_big_crowded and rw_bs < self._bs_deblend_limit and rw_sf < self._sf_deblend_limit):
+                            log.info("The second round of segmentation images still contain big islands or a\n"
+                                     "large source fraction of segments.")
+                            log.info("Proceeding as the time to deblend should be nominal.")
+                            self.kernel = rw2d_kernel
+                            segm_img = copy.deepcopy(rw_segm_img)
+                            del rw_segm_img
+                        # The segmentation image is problematic and the big island/source fraction limits are exceeded,
+                        # so deblending could take days, and the results would not be viable in any case.
+                        else:
+                            log.info("")
+                            log.info("The second round of segmentation images still contain big islands or a\n"
+                                     "large source fraction of segments.")
+                            log.info("The segmentation algorithm is unable to continue.")
```

Comment:
Where will the log message be generated that tells the user that no segmentation catalog will be generated?  It would seem that here would be appropriate if it is not being reported elsewhere.

Reply:
Added the warning that no segmentation catalog will be produced.  Also, changed some  of the former "info" log messages to "warning" log messages.